### PR TITLE
chore(deps): add xUnit console runner

### PR DESCRIPTION
**Pull request by BrainCrumbz**
* chore(tools): add xUnit console runner
* chore(deps): upgrade xUnit to 2.1.0
* chore(deps): upgrade Shouldly to 2.8.2
* chore(tools): upgrade solution to VS 2013

### CODE REVIEW

File: src/Reactive.EventAggregator.sln [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
```
Comment (shiftkey) [inline src/Reactive.EventAggregator.sln]: I'm fine with bringing this up to VS2015, if that helps...

**Comment by shiftkey** [comment]
There's a minor package restore issue as part of the build process (which seems to then trigger a MSBuild warning): https://ci.appveyor.com/project/shiftkey/reactive-eventaggregator/build/1.1.37#L33

Not sure if you're worried about that.

**Comment by BrainCrumbz** [comment]
Retitled PR, as the main need was to actually be able to run tests from console. Upgrading xUnit was just a needed step in the middle.

**Comment by BrainCrumbz** [comment]
Re. the restore issue, I think it is due to the included nuget executable in *.nuget* folder, which by now is an old version.

The real solution to that would be to drop local nuget.exe and .nuget folder altogether, as that approach is discouraged by now. Don't remember though what happens to *build.ps1*.

If it's ok for you, I'd go for a merge, and possibly fix this issue in a separate commit/branch from you/my team/whoever.

**Comment by shiftkey** [comment]
@BrainCrumbz thanks for this!

**Comment by BrainCrumbz** [comment]
This PR is just instrumental to the one we were mainly interested in, about upgrading Rx. Which is under way.

**Pull request merged**